code/__DEFINES/mode.dm

```diff
@@ -39,7 +39,7 @@
 #define BE_ALIEN		1
 #define BE_QUEEN		2
 #define BE_SURVIVOR		4
-#define BE_RESPONDER	8
+#define BE_DEATHMATCH	(1 << 3)
 #define BE_PREDATOR		16
```

Comment:
I'd have thought you'd change all these following the same logic...
```suggestion
#define BE_PREDATOR		(1 << 4)
```
And all...

Reply:
B-but, that takes *effort*! /s